microraiden/microraiden/requests/__init__.py

```diff
@@ -3,15 +3,16 @@
 from requests import Response
 from web3 import Web3
 
-from microraiden import DefaultHTTPClient, Client
+from microraiden import DefaultHTTPClient, Client, HTTPClient
 from microraiden.config import CHANNEL_MANAGER_ADDRESS, CONTRACT_METADATA
 from microraiden.contract_proxy import ChannelContractProxy, ContractProxy
 
-client = None
-http_client = None
+_http_client = None
 
 
 def init(
+    http_client: HTTPClient = None,
+    client_: Client = None,
```

Comment:
underscore at the end is ugly

Reply:
Agreed, will fix. Looks like an earlier version.